spec/controllers/recordings_controller_spec.rb

```diff
@@ -3,8 +3,10 @@
 require 'rails_helper'
 
 RSpec.describe Api::V1::RecordingsController, type: :controller do
-  let(:user) { create(:user) }
-  let(:user_with_manage_recordings_permission) { create(:user, :with_manage_recordings_permission) }
+  let(:user) { create(:user, :with_create_room_permission) }
+  let(:user_with_manage_recordings_permission_and_create_room_permission) do
+    create(:user, :with_manage_recordings_permission, :with_create_room_permission)
+  end
```

Comment:
The createRoom permission is so restrictive, maybe we should just enable it by default with every other permissions?
Else we will have to do this for all of our specs pretty much, no?
Maybe we could have a negative logic trait `:without_create_room_permission` instead and use it when we need to test the createRoom permission.

Reply:
Something like:
```
  let(:user_with_manage_recordings_permission) do
    create(:user, :with_manage_recordings_permission, :with_create_room_permission)
  end
```

And if needed:
```
  let(:user_with_manage_recordings_permission) do
    create(:user, :with_manage_recordings_permission, :without_create_room_permission)
  end
```

Reply:
Actually, maybe there is a way to treat a user without the createRoom permission almost like an unauthorised user instead?

Reply:
Yeah, by default users should have `create room` enabled. We can do it in an `after_create` in the factory